components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1789,7 +1791,7 @@ def ports(self, args, config):
             " regenerated. Use the omero.ports.xxx configuration properties"
             " instead.")
 
-    @admin_only()
+    @admin_only(AdminPrivilegeReadSession)
```

Comment:
No light admin should have `ReadSession` and webadmin provides no way to give it to a light admin.